pages.ne/common/g++.md

```diff
@@ -0,0 +1,29 @@
+# g++
+
+> C++ स्रोत फाइलहरू कम्पाइल गर्छ।
+> GCC को भाग (GNU कम्पाइलर संग्रह).
+> थप जानकारी: <https://gcc.gnu.org>.
+
+- स्रोत कोड फाइललाई कार्यान्वयनयोग्य बाइनरीमा कम्पाइल गर्नुहोस्:
+
+`g++ {{स्रोतफाइल/को/पथ}} -o {{आउटपुट/को/पथ}}`
```

Comment:
```suggestion
`g++ {{स्रोतफाइल.cpp/को/पथ}} -o {{आउटपुट/को/पथ}}`
```

If possible can you mention the executable too in the output path?
Update this throughout all pages in this PR.